var/spack/repos/builtin/packages/squashfuse/package.py

```diff
@@ -19,6 +19,13 @@ class Squashfuse(AutotoolsPackage):
 
     version("master", branch="master")
     version("0.5.0", sha256="d7602c7a3b1d0512764547d27cb8cc99d1b21181e1c9819e76461ee96c2ab4d9")
+    version("0.4.0", sha256="646e31449b7914d2404933aea88f8d5f72c5d135d7deae3370ccb394c40d114a")
+    version("0.2.0", sha256="e8eea1b013b41d0a320e5a07b131bc70df14e6b3f2d3a849bdee66d100186f4f")
+    version(
+        "0.1.105",
```

Comment:
This one does not contain configure scripts, so you need to add `@0.1.105,master` or smth to all those autotools deps.